frontend/src/components/SignIn/SignIn.css

```diff
@@ -37,3 +37,7 @@
   margin-right: 3%;
   margin-bottom: 1%;
 }
+
+.errorValidation {
+  color: red;
```

Comment:
Good idea changing the color of the text! Makes the error seem more apparent.

Reply:
This looks good, but maybe we want to delete the `.errorMessage` CSS since we're not using that anymore?